mlflow/sklearn.py

```diff
@@ -48,25 +61,36 @@ def save_model(sk_model, path, conda_env=None, mlflow_model=Model()):
     >>> sk_path_dir = ...
     >>> mlflow.sklearn.save_model(sk_model, sk_path_dir)
```

Comment:
you might want to add an example here about non-default serialization format.

Reply:
Added an example! Thanks!